Extension/src/LanguageServer/client.ts

```diff
@@ -2651,20 +2636,18 @@
                 next: next
             };
 
-            this.languageClient.sendRequest(GoToDirectiveInGroupRequest, params)
```

Comment:
>sendRequest [](http://example.com/codeflow?start=32&length=11)

Hmm... I thought all `sendRequest` calls were supposed to happen in the context of one of the `whenReady` functions.  Something for us to revisit @colengms.